### PR TITLE
Changed task.py fixed indentation and other changes

### PR DESCRIPTION
**Pull request by shriyanka**
Fixed indentation
added permission_classes in @detail_route

### CODE REVIEW

File: crowdsourcing/serializers/task.py [hunk 1]
```diff
@@ -1,17 +1,39 @@
...
-        read_only_fields = ('module', 'statuses', 'created_timestamp', 'last_updated')
+  deleted = serializers.BooleanField(read_only=True)
+  status=serializers.IntegerField()
+  created_timestamp = serializers.DateTimeField(read_only=True)
 
+  def create(self, **kwargs):
+    task = models.Task.objects.create(deleted = False,**self.validated_data)
+    worker = validated_data.pop('Worker')
+    for t in task:
+      models.TaskWorker.objects.create(task=t,worker=worker)
```
Comment (dmorina) [inline crowdsourcing/serializers/task.py]: tasks are to be generated when the module is created which makes the CUD obsolete but we will accept this for now

File: crowdsourcing/serializers/task.py [hunk 2]
```diff
@@ -1,17 +1,39 @@
...
-        model = models.Task
-        fields = ('module', 'statuses', 'status', 'deleted', 'created_timestamp', 'last_updated')
-        read_only_fields = ('module', 'statuses', 'created_timestamp', 'last_updated')
+  deleted = serializers.BooleanField(read_only=True)
+  status=serializers.IntegerField()
+  created_timestamp = serializers.DateTimeField(read_only=True)
 
+  def create(self, **kwargs):
+    task = models.Task.objects.create(deleted = False,**self.validated_data)
+    worker = validated_data.pop('Worker')
```
Comment (dmorina) [inline crowdsourcing/serializers/task.py]: where is this defined?

File: crowdsourcing/viewsets/task.py [hunk 3]
```diff
@@ -1,15 +1,43 @@
...
+from django.shortcuts import get_object_or_404
+from crowdsourcing.permissions.project import IsProjectCollaborator
 
 class TaskViewSet(viewsets.ModelViewSet):
-    from crowdsourcing.models import Task
-    queryset = Task.objects.all()
-    serializer_class = TaskSerializer
-
+  from crowdsourcing.models import Task
+  queryset = Task.objects.all()
```
Comment (dmorina) [inline crowdsourcing/viewsets/task.py]: missing import?

**Comment by dmorina** [comment]
please test your code before submitting at least using CURL for the api part

File: crowdsourcing/viewsets/task.py (hunk 3 above)
Comment (shriyanka) [inline crowdsourcing/viewsets/task.py]: line 12 has an import..

**Pull request merged**